src/modes/cutscene_world.cpp

```diff
@@ -297,28 +297,32 @@ void CutsceneWorld::update(int ticks)
     {
         if (curr->getType() == "cutscene_camera")
         {
-            scene::ISceneNode* anchorNode = curr->getPresentation<TrackObjectPresentationEmpty>()->getNode();
-            m_camera->setPosition(anchorNode->getPosition());
-            m_camera->updateAbsolutePosition();
-
-            core::vector3df rot = anchorNode->getRotation();
-            Vec3 rot2(rot);
-            rot2.setPitch(rot2.getPitch() + 90.0f);
-            m_camera->setRotation(rot2.toIrrVector());
-
-            irr::core::vector3df up(0.0f, 0.0f, 1.0f);
-            irr::core::matrix4 matrix = anchorNode->getAbsoluteTransformation();
-            matrix.rotateVect(up);
-            m_camera->setUpVector(up);
-
-            SFXManager::get()->positionListener(m_camera->getAbsolutePosition(),
-                                          m_camera->getTarget() -
-                                            m_camera->getAbsolutePosition(),
-                                            Vec3(0,1,0));
-
+            Camera *camera = Camera::getActiveCamera();
+            if (camera && camera->getType() != Camera::CM_TYPE_FPS)
```

Comment:
maybe checking getType() == CM_TYPE_NORMAL would be more future-proof? It's weird to single out FPS cameras like that

Reply:
Fixed and force pushed.